packages/react-instantsearch/src/widgets/Range/__snapshots__/RangeRatings.test.js.snap

```diff
@@ -3,6 +3,7 @@ exports[`Range applies translations 1`] = `
   className="RangeRatings">
   <a
     className="RangeRatings__link"
+    href="#"
```

Comment:
You mean in the test file having a createURL functions a little be smarter than this : createURL={() => '#'} ?